### PR TITLE
Refactor namespace form to resource form

### PR DESCRIPTION
**Pull request by wchrapka**
**Description**

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Predefined/Create/Namespaces/MemoryQuotas.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  const selectedUnit = units.includes(unit) ? unit : '';
+
+  const setValue = val => {
+    jp.value(container, propertyPath, val);
+    setContainer({ ...container });
+  };
+
+  return (
+    <FormItem>
+      <FormLabel required>{label}</FormLabel>
```
Comment (dariadomagala) [inline core-ui/src/components/Predefined/Create/Namespaces/MemoryQuotas.js:34]: You can use the FormField component with `Number` type. Please check the `Issuers` create form as an example

File: core-ui/src/components/Lambdas/config.js [hunk 2]
```diff
@@ -115,13 +115,23 @@ const defaultConfig = {
     cpu: '200m',
   },
   namespaceContainerLimitsPreset: {
+    S: { max: '550Mi', default: '250Mi', defaultRequest: '16Mi' },
```
Comment (dariadomagala) [inline core-ui/src/components/Lambdas/config.js:118]: It's lambdas config, please move the namespace presets to the namespaces folder.

**Review by dariadomagala — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -1056,7 +1056,12 @@ namespaces:
   create-modal:
     apply-limits: Apply limits per container
     apply-memory-quotas: Apply total memory quotas
+    container-limits: Container limits
+    create-limit-range: Create Limit Range
```
Comment (grego952) [inline core/src/i18n/en.yaml:1060]:
```suggestion
    create-limit-range: Create limit range
```

File: core/src/i18n/en.yaml [hunk 4]
```diff
@@ -1056,7 +1056,12 @@ namespaces:
...
+    create-resource-quota: Create Resource Quota
```
Comment (grego952) [inline core/src/i18n/en.yaml:1061]:
```suggestion
    create-resource-quota: Create resource quota
```

File: core/src/i18n/en.yaml [hunk 5]
```diff
@@ -1056,7 +1056,12 @@ namespaces:
...
     disable-sidecar: Disable sidecar injection
+    memory-limits: Memory limit
```
Comment (grego952) [inline core/src/i18n/en.yaml:1063]:
```suggestion
    memory-limits: Memory limits
```
Shouldn't we also use plural here?

**Review by majakurcius — APPROVED** [review]
Please make sure you add Description to your PRs. It's there for a reason ;-)

**Review by dariadomagala — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**